snuba/settings/__init__.py

```diff
@@ -28,6 +28,16 @@
 # Clickhouse Options
 CLICKHOUSE_MAX_POOL_SIZE = 25
 
+SENTRY_LOGICAL_PARTITIONS = 256
+LOCAL_PHYSICAL_SLICES = 1
```

Comment:
Can you add comments around the usage of these settings?

Reply:
as I said above, we really shouldn't change `SENTRY_LOGICAL_PARTITIONS` anywhere. I can add a doc for `LOCAL_PHYSICAL_SLICES`, though